extract/tests/zalando/t-shirt_test.py

```diff
@@ -6,10 +6,10 @@
 
 
 def test_zalando_basic() -> None:
-    timestamp = "2022-02-17 12:49:00"
+    timestamp = "2022-04-22 12:49:00"
     url = "https://www.zalando.mock/"
     merchant = "zalando"
-    file_name = "t-shirt.html"
+    file_name = "t-shirt-new.html"
```

Comment:
Do we use the "old" html? If yes, maybe use postfix "2"? If no, I would simply replace it.

Reply:
No, I didn't know if we want to keep it, but it will be in git if we want to use it at some time, so I will delete the old ones :)